src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({ name }) {
+  return (
+    <div>
+      {name}
+    </div>
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+};
```

Comment:
let's try to add default props, I trust you can so this before merging;

example: 
```
Button.defaultProps = {
  name: '',
};
```